### PR TITLE
Add Stack Overflow as communication channel

### LINKED ISSUE #923: Add Stackoverflow tag for Hawkbit
Hi folks,

today I discussed with @schabdo about having a hawkbit tag for Stackoverflow. Its a great plattform and it would be cool to be able to give people support there (especially as its pretty visible for devs / via google).

What do others think?

### PR DESCRIPTION
**Pull request by schabdo**
Resolves #923

### CODE REVIEW

File: docs/content/community.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Questions? Problems? Searching for a quick response from the team behind hawkBit
 ## Mailing List
 A great way to stay up to date with hawkBit activity is to subscribe to the Mailing list provided by Eclipse. Sign up for the mailing list [here](https://dev.eclipse.org/mailman/listinfo/hawkbit-dev).
 
+## Stack Overflow
+
+Find existing questions or raise a new one tagged with `eclipse-hawkbit` [here](https://stackoverflow.com/questions/tagged/eclipse-hawkbit).
```
Comment (laverman) [inline docs/content/community.md]:
```suggestion
We want to use Stack Overflow for questions regarding hawkBit. So if you have any, please have a look [here](https://stackoverflow.com/questions/tagged/eclipse-hawkbit). If you can't find it - raise it and tag it with `eclipse-hawkbit`. Hence, others will benefit from your question, too.
```

Comment (StefanKlt) [reply docs/content/community.md]:
```suggestion
Visit [stackoverflow.com](https://stackoverflow.com/questions/tagged/eclipse-hawkbit) to find and raise questions tagged with `eclipse-hawkbit`.
```

Comment (schabdo) [reply docs/content/community.md]: @laverman @StefanKlt thanks for your feedback. What about this one?

**Comment by laverman** [comment]
Even better now :-)

**Pull request merged**